src/AzureTopicConfigurationStrucuture.json

```diff
@@ -23,6 +23,10 @@
     "topicName": "cohort_approved_by_employer",
     "subscription": [ "Task_CohortApproved" ]
   },
+  {
+    "topicName": "commitment_requires_approval_by_transfer_sender",
```

Comment:
Can we call this cohort instead of commitment, to be consistent with others, and maybe word it a little differently, such as "cohort_sent_to_transfer_sender_for_approval". Or something like that. But not that. Better than that.

Reply:
changed to 'cohort_approval_by_transfer_sender_requested'